cmd/noms/splore/js2go.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+import sys
+
+def main():
+    if len(sys.argv) != 4:
+        print('usage: %s <source> <package> <varname>' % (sys.argv[0]))
+        return
+
+    jsf, pkg, varname = sys.argv[1:]
+
+    sys.stdout.write('package %s\n\n' % (pkg,))
+    sys.stdout.write('const %s = "' % (varname,))
+    with open(jsf, 'r') as js:
+        sys.stdout.write(
+                js.read()
+                .replace('\\', '\\\\')
+                .replace('"', '\\"')
+                .replace('\n', '" +\n"'))
+    sys.stdout.write('"')
```

Comment:
> unless we already require it somewhere

Yeah, we have Python elsewhere in the code (for most scrips).

> Not understanding why this script is necessary

I did spend a small amount of time researching how to do this. Basically I need to build resources into a .go binary, because distributing it as a separate resource is a non-starter. AFAIK there is no general JS -> Go way of doing this (and besides this script is super simple).

Perhaps a more idiomatic way of doing these things is to use a `go generate` script but given we need to use npm/yarn for building the JS anyway, it seemed better to stick with only a single build system.